### PR TITLE
Add Yuan Zeng philadelphia recommendations

### LINKED ISSUE #14: Add YuanZeng Philadelphia file
- [x] Create a **branch**
- [x] Add file/commits
- [x] Open a PR
- [x] Have a discussion about the PR (COLLABORATE)
- [x] Make edits
- [x] Get :shipit: approval
- [x] Merge PR

### PR DESCRIPTION
**Pull request by YuanZeng**
closes #14 

Hey, @redlands , @tingleeGIS  looks good? :smile: 

cc: @beardofedu

### CODE REVIEW

File: YuanZeng-Philadelphia.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#Philadelphia, PA
+
+###Places to Eat
+- Percy Street BBQ
+- Han Dynasty
+
+###Places to Go
```
Comment (redlands) [inline YuanZeng-Philadelphia.md]: do you :surfer:

**Comment by redlands** [comment]
looks great!

**Comment by tingleeGIS** [comment]
Please recommend place to :snowboarder:

**Comment by redlands** [comment]
from google: Huntington Beach Surf City USA is the capital of surfing in California.

**Comment by YuanZeng** [comment]
@tingleeGIS @redlands Added ski resort, does this look good to :shipit: ?

**Pull request merged**